sentry-ruby/lib/sentry/interfaces/request.rb

```diff
@@ -123,13 +123,14 @@ def is_skippable_header?(key)
         !(key.start_with?('HTTP_') || CONTENT_HEADERS.include?(key))
     end
 
-    # Rack adds in an incorrect HTTP_VERSION key, which causes downstream
+    # In versions < 3, Rack adds in an incorrect HTTP_VERSION key, which causes downstream
     # to think this is a Version header. Instead, this is mapped to
     # env['SERVER_PROTOCOL']. But we don't want to ignore a valid header
     # if the request has legitimately sent a Version header themselves.
     # See: https://github.com/rack/rack/blob/028438f/lib/rack/handler/cgi.rb#L29
-    # NOTE: This will be removed in version 3.0+
     def is_server_protocol?(key, value, protocol_version)
+      return false if ::Rack.release.start_with?('3')
```

Comment:
This will be wrong in Rack 4, no?

Reply:
Rack has a major bump once in 5 years or so, think of it like WSGI, it doesn't change much.

Reply:
But yea, i guess I could do an int here, thx!